style.css

```diff
@@ -0,0 +1,153 @@
+li,
```

Comment:
- [ ] @Lasten-Ernest kindly change your CSS name to match the project requirement  as shown below 
![image](https://user-images.githubusercontent.com/61408860/221111610-42cb94c9-052c-4c79-8cc7-673cf8d10f52.png)